CODEOWNERS

```
@@ -0,0 +1,2 @@
+/src/ansys/templates/python/solution/ @ansys/Solution-Maintainers
+/src/ansys/templates/python/osl_solution @ansys/Solution-Maintainers
```

I **think** this is functionally equivalent to the line above, but the inconsistency with the trailing `/` is triggering me 😨 
```suggestion
/src/ansys/templates/python/osl_solution/ @ansys/Solution-Maintainers
```